Add information about Asynchronous Storage Pod idling #1459 - review (#1461)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/proc_using-different-types-of-storage.adoc

a user can simultaneously  run only 1 workspace with asynchronous storage

To configure asynchronous storage for your workspace add the following to workspace configuration:

> +

Reviewer: No need -- it's not a list.
```suggestion

```

[source,yaml]
----
attributes:
 asyncPersist: 'true'
 persistVolumes: 'false'
----

> Also, the two new properties were added to `che.properties` that can be used to configure behavior of a client like Che UserDashboard:

Reviewer: Useless sentence.
```suggestion
```

Reviewer 2: To add, I'd avoid using "the new X" in the docs, since eventually it'll no longer be new.

Reviewer: @MichalMaler, please, remove the sentence.

+1 to Angel's remark -- that holds true everywhere.

`che.workspace.storage.available_types`: this configuration property defines available values for storage types that clients like Dashboard should propose for users during workspace creation/update. Available values: 'persistent’,'ephemeral', 'async'. You can use several type comma separate (e.g. `che.workspace.storage.available_types=persistent,ephemeral,async`)

`che.workspace.storage.preferred_type`: this configuration property defines a default value for storage type that clients like Dashboard should propose for users during workspace creation/update. The 'async' value not recommended as a default type since it's still experimental (e.g. `che.workspace.storage.preferred_type=persistent`)

The Storage Type switcher is available on the *Create Custom Workspace* page of the User Dashboard:

> +

Reviewer: No need -- not a list.
```suggestion

```

image::workspaces/create-custom-ws-storage-type-2.png[link="{imagesdir}/workspaces/create-custom-ws-storage-type-2.png"]

> +

Reviewer: dtto
```suggestion

```

image::workspaces/create-custom-ws-storage-type.png[link="{imagesdir}/workspaces/create-custom-ws-storage-type.png"]

.Idling Asynchronous Storage Pods

> {prod-shor} can shutdown Asynchronous Storage Pod if not used for a configured period of time. 

Reviewer: ```suggestion
{prod-short} can shut down Asynchronous Storage Pod if it is not used for a configured period of time. 
```
IBM Style Guide p. 362.

> {prod-shor} can shutdown Asynchronous Storage Pod if not used for a configured period of time. 

Reviewer 2: To add to Robert's comment

> can shut down **the** Asynchronous Storage Pod if **it is** not used

Author: done
